### PR TITLE
Plane: fixed use of VTOL control for missions

### PR DESCRIPTION
**Pull request by tridge**
horizontal speed was not being setup when doing waypoint missions in VTOL mode

### CODE REVIEW

File: ArduPlane/quadplane.cpp [hunk 1]
```diff
@@ -2486,6 +2486,10 @@ void QuadPlane::setup_target_position(void)
     // setup vertical speed and acceleration
     pos_control->set_max_speed_z(-pilot_velocity_z_max, pilot_velocity_z_max);
     pos_control->set_max_accel_z(pilot_accel_z);
+
+    // setup horizontal speed and acceleration
+    pos_control->set_max_speed_xy(wp_nav->get_default_speed_xy());
+    pos_control->set_max_accel_xy(wp_nav->get_wp_acceleration());
```
Comment (khancyr) [inline ArduPlane/quadplane.cpp:2492]: don't you need 
```
// initialise velocity controller
    pos_control->init_vel_controller_xyz();

?
```

Comment (tridge) [reply ArduPlane/quadplane.cpp:2492]: I tend to follow what copter does, and the init_vel_controller_xyz() call is only used in GUIDED mode

Comment (peterbarker) [reply ArduPlane/quadplane.cpp:2492]: The velocity controller isn't used in quadplane; there's no update to `update_vel_controller_xyz`.  No need to init that controller, then.  The xy-controller is used on quadplane, and it is initialised.

Comment (khancyr) [reply ArduPlane/quadplane.cpp:2492]: ok! my bad

**Pull request merged**